addons/project_timesheet_holidays/tests/test_timesheet_holidays.py

```diff
@@ -85,7 +83,7 @@ def test_validate_with_timesheet(self):
             'number_of_days': number_of_days,
         })
         holiday.sudo().action_validate()
-        self.assertEquals(len(holiday.timesheet_ids), number_of_days, 'Number of generated timesheets should be the same as the leave duration (1 per day between %s and %s)' % (fields.Datetime.to_string(self.leave_start_datetime), fields.Datetime.to_string(self.leave_end_datetime)))
+        self.assertEquals(len(holiday.timesheet_ids), number_of_days, 'Number of generated timesheets should be the same as the leave duration (1 per day between %s and %s)' % (self.leave_start_datetime, self.leave_end_datetime))
```

Comment:
Other thing that should be in an isolated PR

Reply:
😩

Reply:
The concept is simple: one change per PR :wink:

Reply:
#30872